src/RoadRegistry.Wms.Projections/IStreetNameCache.cs

```diff
@@ -14,48 +11,4 @@ public interface IStreetNameCache
         Task<long> GetMaxPositionAsync(CancellationToken token);
         Task<Dictionary<int, string>> GetStreetNamesByIdAsync(IEnumerable<int> streetNameIds, CancellationToken token);
     }
-
-    public class StreetNameCache : IStreetNameCache
```

Comment:
One file, mostly one type - helps with discovering.

Reply:
Fine with me, but what are the exceptions?

Reply:
Delegates, private types, mostly.